contracts/deploy.sh

```diff
@@ -82,6 +82,8 @@ run_script script/deploy/DeployL1BridgeContracts.s.sol DeployL1BridgeContracts
 run_script script/deploy/InitializeL1BridgeContracts.s.sol InitializeL1BridgeContracts
 run_script script/deploy/InitializeL2BridgeContracts.s.sol InitializeL2BridgeContracts
 run_script script/deploy/InitializeL1T1Owner.s.sol InitializeL1T1Owner
+run_script script/deploy/DeployL2t1XChainReader.s.sol DeployL2t1XChainReader
+run_script script/deploy/DeployPR1t1XChainReader.s.sol DeployPR1t1XChainReader
```

Comment:
Shall we add the other deploy scripts for ERC7683?

Reply:
I don't think we should require 7683 to be deployed on each release